ci/scripts/install_sccache.sh

```diff
@@ -39,30 +39,30 @@ SCCACHE_URL="https://github.com/mozilla/sccache/releases/download/v$VERSION/scca
 SCCACHE_ARCHIVE=sccache.tar.gz
 
 # Download archive and checksum
-curl -L $SCCACHE_URL --output $SCCACHE_ARCHIVE
-curl -L $SCCACHE_URL.sha256 --output $SCCACHE_ARCHIVE.sha256
+curl -L "$SCCACHE_URL" --output $SCCACHE_ARCHIVE
+curl -L "${SCCACHE_URL}.sha256" --output $SCCACHE_ARCHIVE.sha256
 echo "  $SCCACHE_ARCHIVE" >> $SCCACHE_ARCHIVE.sha256
 
-SHA_ARGS="--check --status"
+SHA_ARGS=(--check --status)
 
 # Busybox sha256sum uses different flags
 if sha256sum --version 2>&1 | grep -q BusyBox; then
-  SHA_ARGS="-sc"
+  SHA_ARGS+=(-sc)
```

Comment:
Oh this would have just overwritten each other before, nice catch!

Reply:
Hmm. I think that we need to overwrite `--check --status` with `-sc` because BusyBox based `sha256sum` doesn't accept `--check --status`.

Reply:
@assignUser @kou Thank you for your comment. I changed to `SHA_ARGS="-sc"`.